site/content/docs/seo.md

```diff
@@ -0,0 +1,47 @@
+---
+title: SEO Configuration
+description: How to set up SEO in flowershow
+---
+
+Flowershow supports adding standard metadata to your pages to improve social media sharing and search engine discovery.
+
+Metadata values are set for individual pages from their frontmatter title and description. Additionally, you can add a default image and description that will be used for the pages if they are not set explicitly.
+
+Flowershow uses the [NextSeo]( https://github.com/garmeeh/next-seo) plugin that helps to add SEO properties to the page and is a fairly easy setup to follow. 
```

Comment:
Maybe? As a callout?

Flowershow uses the NextSEO plugin under the hood, so you can also add other metadata to your pages. See their official [documentation]( https://github.com/garmeeh/next-seo) to learn more.